QtAwesome/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+project(QtAwesome)
+
+add_library(QtAwesome
```

Comment:
Looks like you forgot the root cmakelists with find_package or you forgot find_package here